src/ert/ensemble_evaluator/snapshot.py

```diff
@@ -414,7 +495,7 @@ def aggregate_real_states(self) -> typing.Dict[str, int]:
             states[status] += 1
         return states
 
-    def data(self) -> Mapping[str, Any]:
+    def data(self) -> Dict[str, Any]:
```

Comment:
Why we can't use Mapping? When using Dict it implies that we want to change the content of it - is this the case?

Reply:
Is this for all the Dict[] occurences? Strictly, we return a dictionary of partial snapshot by calling to_dict(), but the user might not care that this is a specifically a dictionary as long as it is something that implements/extends mapping. Should we specify it using the specific type or the abstract?